src/ty/mod.rs

```diff
@@ -485,3 +497,58 @@ where
         &self.type_param
     }
 }
+
+/// Type describing a [`bitvec::vec::BitVec`].
+#[cfg(feature = "bit-vec")]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(any(feature = "std", feature = "decode"), derive(scale::Decode))]
+#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Debug)]
+pub struct TypeDefBitVec<T: Form = MetaForm> {
```

Comment:
I'd rename this to `TypeDefBitSequence` to match with `Sequence` type definition.